### DISCUSSION TITLE
build bootstrap with tailwindcss

### DISCUSSION THREAD

**Discussion by CFarhad** [twbs/bootstrap#35700] [Q&A] [answered]
hello guys.

i wanna mix bootstrap and tailwind for build new bootstrap component and my idea is replace tailwind `theme()` to bootstrap variables. 
example for: `$primary: theme('colors.blue.500')`

but this is not working.
i use laravel-mix for build tailwindcss and sass

```bash
SassError: $color: theme("colors.blue.500") is not a color.
   ╷
37 │   @return red($value), green($value), blue($value);
   │           ^^^^^^^^^^^
   ╵
  node_modules\bootstrap\scss\_functions.scss 37:11  to-rgb()
  node_modules\bootstrap\scss\_functions.scss 61:36  map-loop()
  node_modules\bootstrap\scss\_variables.scss 94:20  @import
  src\tailboot.scss 8:9                              root stylesheet
```

who can help me or who have better idea.

**Comment by ascensus-mdb** [comment, marked as answer]
Hi,

there is a ready to use solution for mixing tailwind and bootstrap toghether 

https://tailwind-elements.com/
